src/calculateRentalCost.js

```diff
@@ -21,14 +21,22 @@
  *  - calculateRentalCost(1) === 40
  *  - calculateRentalCost(3) === 100
  *  - calculateRentalCost(7) === 230
- *
- * @param {number} days
- *
- * @return {number}
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const payment = 40;
+  const lowDiscount = 20;
+  const highDiscount = 50;
+
+  if (days >= 7) {
+    return days * payment - highDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
You can remove the condition `days < 7` since it is redundant. If the code reaches this point, it implies that `days < 7` is already true. 😊